src/covid_model_deaths_spline/cluster.py

```diff
@@ -12,10 +12,10 @@
 
 PROJECT = 'proj_dq'
 QUEUE = 'd.q'
-F_MEM = '3.0G'
-F_THREAD = '20'
+F_MEM = '5.0G'
+F_THREAD = '13'
```

Comment:
79 % 13 = 1, so this leaves only one unused thread per machine if the d.q is empty.